Packs/SafeNet_Trusted_Access/.pack-ignore

```diff
@@ -1,5 +1,4 @@
-[file:SafeNet_Trusted_Access_–_Add_to_Unusual_Activity_Group.yml]
+[file:playbook-SafeNet_Trusted_Access_-_Add_to_Unusual_Activity_Group.yml]
+ignore=PB106
```

Comment:
Why you are ignoring this error?

Reply:
As discussed with Kobbi, this is affecting the usecase functionality. So as suggested, added this under the .pack-ignore file.
For our previous version pack, I am not able to see our playbook readme like https://xsoar.pan.dev/docs/reference/playbooks/abuse-inbox-management-detect--respond 
Please check and let us know the exact reason for the same.

Reply:
This error indicates that the Playbook is bound to a specific Integration instance. I informed Joshi that this is not best practice (that the Integration instance name is used as an input to a Playbook). 

Joshi informed me that they have a guide to that instructs users to specify the instance name.

Reply:
Thank you Kobbi.